### DISCUSSION TITLE
Can I get custom field's display value instead of the tag value in a ticket

### DISCUSSION THREAD

**Discussion by HQ-Panda** [cloudbees/zendesk-java-client#452] [Q&A] [answered]
I get the value of a custom field by:
```
Ticket ticket = zendesk.getTicket(ticketId);
for(CustomField customoField : ticket.getCustomFields()){
String value = customoField .getValue()[0];
}
```
However, the value obtained by this method is the tag value
What should I do if I want to get the display value

**Comment by aheritier** [comment, marked as answer]
Hi @xg578962913 

Zendesk APIs requires that you use the Ticket Fields API : https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_fields/#show-ticket-field

In java: https://github.com/cloudbees-oss/zendesk-java-client/blob/master/src/main/java/org/zendesk/client/v2/Zendesk.java#L603-L610

You should probably cache your TicketsFields definition and then from the customField you get the Field and you use the mapping 
https://github.com/cloudbees-oss/zendesk-java-client/blob/master/src/main/java/org/zendesk/client/v2/model/Field.java#L38

**Reply by HQ-Panda** [reply]
It worked，Thank you  for your answer
